examples/preact/.storybook/main.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  framework: '@storybook/preact',
+  stories: ['../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-essentials'],
+  core: {
+    builder: '@storybook/builder-vite',
+    // we don't want to muck up the data when we're working on the builder
+    disableTelemetry: true,
+  },
+  features: {
+    storyStoreV7: false,
```

Comment:
Just curious, is this set to false because it has a problem, or because you copied from another example that had it set to false?

Reply:
Copied from the [react example](https://github.com/storybookjs/builder-vite/blob/3d4b66b789ae740b1606ed7013ccbb90ee8e4f3d/examples/react/.storybook/main.js#L11). My thinking was to try to keep the two examples nearly identical to verify the preact framework detection.